spec/facility_spec.rb

```diff
@@ -22,5 +27,64 @@
       @facility.add_service('Vehicle Registration')
       expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
     end
+    it 'can register vehicle' do
+      @facility.add_service('Vehicle Registration')
+      expect(@facility.registered_vehicles).to eq([])
+      expect(@facility.collected_fees).to eq(0)
+      @facility.register_vehicle(@cruz)
+      expect(@cruz.registration_date).to be_within(1.0).of(Time.now)
+      expect(@cruz.plate_type).to eq(:regular)
+      expect(@facility.registered_vehicles).to eq([@cruz])
+      expect(@facility.collected_fees).to eq(100)
+
+    end
+    
+  end
+  describe 'written test' do
+    it 'can take test' do
```

Comment:
Generally it's a better practice to use `true` and `false` instead of `!false` and `!true`. These tests would work the same way with this change.
If you really need to test for any false-y value (in Ruby, the only false-y values are `nil` and `false`), you could do something like `expect(<whatever>).to be_falsey`, or for the opposite, `expect(<whatever>).to be_truthy`